src/app/routes.tsx

```diff
@@ -41,11 +42,18 @@ const dynamicRoutes: IAppRoute[] = [
     exact: true,
     label: 'Recordings',
     path: '/recordings',
-    title: 'Flight Recordings'
+    title: 'Recordings'
   },
+  {
+    component: Events,
+    exact: true,
+    label: 'Events',
+    path: '/events',
```

Comment:
Maybe this is local only has to do with CORs or something? I tested this with container-jfr pointing to your work @ https://github.com/rh-jmc-team/container-jfr/pull/148

If I visit / and click events, it brings me to /events fine

If I take the url from /events (ip-addr:port/events) and visit it in another tab, it shows "Resource not found"

I'd expect it to open up the events page. Any ideas what's going on?

Reply:
Not sure, can't reproduce it. Initially the Events view, as well as the Recordings view, should produce a "Resource not found" because at startup they really aren't defined, because there is no command channel or target connection. But once the application establishes a command channel connection and sees that there is also a target connection then the routes should be made available. That's specifically implemented in `routes.tsx` and seems to work on my end.

Reply:
Huh weird... reproducer steps should be, on:
https://github.com/rh-jmc-team/container-jfr/pull/148
with web-client @ this pr.

$ MAVEN_OPTS="-Dcom.sun.management.jmxremote.port=9091 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false" mvn clean prepare-package exec:java

(Using Google Chrome)
* Visit HTTP service @ e.g. http://192.168.0.27:8181 and target container-jfr
* Click on events to get to: http://192.168.0.27:8181/events
* Copy-paste http://192.168.0.27:8181/events into a new tab and hit enter. See "Resource not found"

I notice if I vist http://192.168.0.27:8181 in a new tab, it is already set to be connected to container-jfr and changes to the target are seen by both tabs. Would visiting Events view directly miss some of the code that gets the target?

Reply:
Ah - I see. It's broken in the way you describe when you do an actual client build and serve it from ContainerJFR, but it works as I intended when using Webpack Dev Server. That's unfortunate. I'll have to do some research to see how to get the web client running and preserve the path to pass to its internal routing. I'll file a separate issue.

Reply:
In other words - this is actually a ContainerJFR webserver problem, not a ContainerJFR Web problem

Reply:
Okay, sounds good. Please resolve this conversation when an issue is filed.

Reply:
#64